moveit_ros/planning_interface/common_planning_interface_objects/src/common_objects.cpp

```diff
@@ -73,22 +73,6 @@ SharedStorage& getSharedStorage()
   return *storage;
 #endif
 }
-
```

Comment:
Why is this being removed?

Reply:
It's not used at all after this PR

Reply:
You mean that usage was removed in the past? I really had to look up where this was used at all, seemed completely unrelated. But I agree that it can be removed.